test/parallel/test-process-versions.js

```diff
@@ -53,10 +60,9 @@ assert.match(
 assert.match(process.versions.modules, /^\d+$/);
 
 if (common.hasCrypto) {
-  const versionRegex = common.hasOpenSSL3 ?
-    // The following also matches a development version of OpenSSL 3.x which
-    // can be in the format '3.0.0-alpha4-dev'. This can be handy when building
-    // and linking against the main development branch of OpenSSL.
+  const versionRegex = common.hasOpenSSL3 ? // The following also matches a development version of OpenSSL 3.x which
+  // can be in the format '3.0.0-alpha4-dev'. This can be handy when building
+  // and linking against the main development branch of OpenSSL.
```

Comment:
```suggestion
  const versionRegex = common.hasOpenSSL3 ?
    // The following also matches a development version of OpenSSL 3.x which
    // can be in the format '3.0.0-alpha4-dev'. This can be handy when building
    // and linking against the main development branch of OpenSSL.
```